lib/mrsk/cli/base.rb

```diff
@@ -1,5 +1,5 @@
 require "thor"
-require "mrsk/sshkit_with_ext"
+require "ssh_kit/backend/abstract"
```

Comment:
I'm not sure if this is the right thing to do here?

We get a warning:

```
❯ bin/test
WARNING: Zeitwerk defines the constant SshKit after the directory

    ..../mrsk/lib/ssh_kit

To prevent that, please configure the loader to ignore it:

    loader.ignore("#{__dir__}/ssh_kit")

Otherwise, there is a flag to silence this warning:

    Zeitwerk::Loader.for_gem(warn_on_extra_files: false)
Run options: --seed 58503

# Running:

............................................................................................

Finished in 0.041901s, 2195.6517 runs/s, 3269.6117 assertions/s.
92 runs, 137 assertions, 0 failures, 0 errors, 0 skips
```